code_generation/analysis_template.cxx

```diff
@@ -33,7 +33,7 @@ int main(int argc, char *argv[]) {
 
     TStopwatch timer;
     timer.Start();
-    ROOT::EnableImplicitMT(1); // for multithreading
+    ROOT::EnableImplicitMT(8); // for multithreading
```

Comment:
keep this as 1 for the default

Reply:
argh, I thought I had checked out this file...